ocf_data_sampler/constants.py

```diff
@@ -110,7 +110,7 @@ def __getitem__(self, key):
     "lcc": 0.44901806116104126,
     "mcc": 0.3288780450820923,
     "prate": 3.108070450252853e-05,
-    "sde": 8.107526082312688e-05,
+    "sd": 0.0006500509,
```

Comment:
Change to "sd" but keep old value. This means PVnet DA model will still work. Retrain Pvnet INtraday using this values.